src/lib/jsonql/promises.js

```diff
@@ -1,5 +1,16 @@
 import { equalsIgnoreCase, slugify } from "@/promisetracker/utils";
 
+const formatLocation = (latlng) => {
+  if (latlng) {
+    const formatted = latlng.split(",");
+    return [parseFloat(formatted[0]), parseFloat(formatted[1])];
+  }
+  if (process.env.DEFAULT_LAT && process.env.DEFAULT_LONG) {
+    return [process.env.DEFAULT_LAT, process.env.DEFAULT_LONG];
+  }
+  return null;
+};
```

Comment:
Works but I'm expecting something like 👇🏽 ... we only send location if we are 💯 sure both `lat` and `lng` are valid numbers.
```suggestion
const formatLocation = (latlng) => {
  const [latStr, lngStr] = latlng?.split(",") ?? [process.env.DEFAULT_LAT, process.env.DEFAULT_LONG]
  if (latStr && lngStr) {
    const lat = parseFloat(latStr);
    const lng = parseFloat(lngStr);
    if (Number.isFinite(lat) && Number.isFinite(lng)) {
      return [lat, lng];
    }
  }
  return null;
};
```